### DISCUSSION TITLE
Direct/dirty deploys with no path parameter supplied should direct deploy all functions

### DISCUSSION THREAD

**Discussion by filmaj** [architect/architect#1286] [Feature Requests]
**Describe the issue**
currently if you provide no path parameter to direct deploy, you will be given an error that says 'no lambdas could be found'

**Expected behavior**
Instead of erroring, deploy all lambdas directly.

**Comment by ryanblock** [comment]
This has some interesting performance considerations and some implementation details to consider – do you still want to do this @filmaj?
